webapp/portlet/src/main/resources/locale/portlet/dlpQuarantine/dlpQuarantine_en.properties

```diff
@@ -8,3 +8,8 @@ documents.dlp.quarantine.author=Author
 documents.dlp.quarantine.previewDownload=Preview / download the document
 documents.dlp.quarantine.validateDoc=Validate the document
 documents.dlp.quarantine.deleteDoc=Delete the document
+
+documents.dlp.message.confirmRestore=Are you sure you want to restore this document?
+documents.dlp.title.confirmRestore=Restore document
+documents.dlp.button.ok=OK
+documents.dlp.button.cancel=Cancel
```

Comment:
this should be modified to stay generic.
Documents is KO. It should be Item or Element